fineftp-server/include/fineftp/server.h

```diff
@@ -52,8 +53,10 @@ namespace fineftp
      * 
      * @param port: The port to start the FTP server on. Defaults to 21.
      * @param host: The host to accept incoming connections from.
+     * @param output: Normal output prints. Defaults to std::cout.
+     * @param error: Error output prints. Defaults to std::cerr.
      */
-    FINEFTP_EXPORT FtpServer(const std::string& address, uint16_t port = 21);
+    FINEFTP_EXPORT FtpServer(const std::string& address, const uint16_t port = 21, std::ostream& output = std::cout, std::ostream& error = std::cerr);
```

Comment:
`port` should not be const, as it is passed by value (will give warnings otherwise)

Reply:
Can you give an example of where you might think it will fail?